### PR TITLE
Syn 676

### PR DESCRIPTION
**Pull request by tbunitrade**
Updates for Contact Us page, delete Downloads move NDS link to bottom URL.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,10 +3,15 @@ wp-content/uploads/
 wp-content/mu-plugins/
 wp-content/themes/syndicode/node_modules/
 wp-content/cache/
+wp-content/plugins/
+wp-content/aiowps_backups/
+.gitignore
```
Comment (LedangNguyen) [inline .gitignore:8]: Remove this line

File: .gitignore [hunk 2]
```diff
@@ -3,10 +3,15 @@ wp-content/uploads/
 wp-content/mu-plugins/
 wp-content/themes/syndicode/node_modules/
 wp-content/cache/
+wp-content/plugins/
```
Comment (LedangNguyen) [inline .gitignore:6]: Remove this line

File: wp-content/themes/syndicode/inc/carbon-fields/blocks/template-services-new/engagement-models.php [hunk 3]
```diff
@@ -19,29 +19,26 @@
...
-				<?php
-				foreach ( $fields['crb_wp_dev_engagement_models_items'] as ['crb_wp_dev_engagement_models_image' => $image, 'crb_wp_dev_engagement_models_title' => $title, 'crb_wp_dev_engagement_models_text' => $text] ) :
-					?>
+				<?foreach ( $fields['crb_wp_dev_engagement_models_items'] as ['crb_wp_dev_engagement_models_image' => $image, 'crb_wp_dev_engagement_models_title' => $title, 'crb_wp_dev_engagement_models_text' => $text] ) : ?>
 					<div class="wp-dev-company-engagement-models__item">
-						<?php echo get_attachment_image( $image ); ?>
+						<?php 
+						$data = get_attachment_image( $image );
+						echo '<pre>';
+						var_dump($data);
```
Comment (LedangNguyen) [inline wp-content/themes/syndicode/inc/carbon-fields/blocks/template-services-new/engagement-models.php:33]: Is it supposed to be here?

**Review by LedangNguyen — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (tbunitrade) [reply .gitignore:8]: OK

**Review by LedangNguyen — COMMENTED** [review]
Do we need Classic Editor?

File: .gitignore (hunk 1 above)
Comment (LedangNguyen) [reply .gitignore:8]: `.gitignore` still there

**Review by LedangNguyen — APPROVED** [review]

**Pull request merged**